specification/databox/resource-manager/Microsoft.DataBox/stable/2020-04-01/examples/ServiceValidateInputs.json

```diff
@@ -0,0 +1,76 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "location": "westus",
+    "api-version": "2020-04-01",
+    "x-ms-client-request-id": [
+      "ee686d79-42c2-4b01-9afe-5e99f4cabfee"
+    ],
+    "accept-language": [
+      "en-US"
+    ],
+    "User-Agent": [
+      "FxVersion/4.6.26614.01",
+      "Microsoft.Azure.Management.DataBox.DataBoxManagementClient/1.2.0"
+    ],
+    "Content-Type": [
+      "application/json; charset=utf-8"
+    ],
+    "Content-Length": [
+      "311"
+    ],
+    "validationRequest": {
+      "validationCategory": "JobCreationValidation",
+      "individualRequestDetails": [
+        {
+          "validationType": "ValidateDataDestinationDetails"
```

Comment:
ValidateDataDestinationDetails is deprecated in the new api versions. Kindly use ValidateDataImportDetails.

Reply:
Okay